### DISCUSSION TITLE
[blocks]: Sidebar

### DISCUSSION THREAD

**Discussion by surafel-99** [shadcn-ui/ui#3434] [Blocks Request]
### Description

Custom and responsive sidebar that is used to build different dashboard and custom websites!

![Sidebar Navigation - Exploration](https://github.com/shadcn-ui/ui/assets/63979787/1c903c38-d23f-48da-b6b3-672bbf848f97)
![Minimal sidebar navigation for dashboard](https://github.com/shadcn-ui/ui/assets/63979787/01b7ad02-eb70-4d5a-ad9c-ffc30675e888)
![Dashboard + Navigation Interface](https://github.com/shadcn-ui/ui/assets/63979787/8d705c15-82cd-4dab-a955-e6439fdea1a1)
![Sidebar Navigation UX_UI](https://github.com/shadcn-ui/ui/assets/63979787/824e1677-fc3a-479f-ab9c-2e93b4153ee1)
![Sidebar navigation](https://github.com/shadcn-ui/ui/assets/63979787/6c7eeae9-4537-4439-ace7-6ce65394fb5e)

**Comment by ishaanw007** [comment]
https://github.com/shadcn-ui/ui/discussions/3652
Please have a look i have implemented

**Reply by gsmatheus** [reply]
Would you be able to share your example by uploading it to a repository?
